cal/cal/static/css/calendaranalytics.css

```diff
@@ -15,6 +15,20 @@ h1, h2, h3, h4, h5, h6, a, p {
 button {
   margin: 2px; }
 
+.section-info {
+  font-size: 14px;
+  background-color: #fff;
+  border: 1px solid #000;
+  border-radius: 5px;
+  padding: 10px;
+  margin: auto;
+  line-height: normal;
+}
+
+.fc-content {
+    cursor: pointer;
+}
+
```

Comment:
Basically, you should never have to modify .css files. Instead, you can take advantage of the superior syntax of sass, including nested selectors and variables B)